<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>741</width>
      <height>491</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Add">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>80</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditType">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTypeKey">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter data to add:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter key to add:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTypeKeyDel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter key to delete:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DirectOrder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PrintInDirectOrder</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDirectOrder">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ReverseOrder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>360</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PrintInReverseOrder</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelReverseOrder">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>360</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="AscendingOrder">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PrintInAscendingOrder</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelAscendingOrder">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>400</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter the vertex whose branch you want to delete:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditTask">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>270</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Task">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>270</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Task</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSearch">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>480</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter key to search:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Search">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>480</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSearchResult">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>520</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>GeoSlab703 MdCn BT</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>The results of search:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
